distribution/tools/launchers/src/main/java/org/opensearch/tools/launchers/SystemJvmOptions.java

```diff
@@ -101,15 +101,15 @@ private static String maybeShowCodeDetailsInExceptionMessages() {
     }
 
     private static String javaLocaleProviders() {
-        /**
-         *  SPI setting is used to allow loading custom CalendarDataProvider
-         *  in jdk8 it has to be loaded from jre/lib/ext,
-         *  in jdk9+ it is already within ES project and on a classpath
-         *
-         *  Due to internationalization enhancements in JDK 9 OpenSearch need to set the provider to COMPAT otherwise time/date
-         *  parsing will break in an incompatible way for some date patterns and locales.
-         *  //TODO COMPAT will be deprecated in at some point, see please https://bugs.openjdk.java.net/browse/JDK-8232906
-         * See also: documentation in <code>server/org.opensearch.common.time.IsoCalendarDataProvider</code>
+        /*
+           SPI setting is used to allow loading custom CalendarDataProvider
+           in jdk8 it has to be loaded from jre/lib/ext,
+           in jdk9+ it is already within ES project and on a classpath
+          <p>
```

Comment:
This is not part of API javadoc and won't be displayed, I think we should remove `<p>` from here

```suggestion
          
```